addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -373,15 +393,47 @@ class AccountTax(models.Model):
                     'l10n_it_kind_exoneration',
                     'l10n_it_law_reference',
                     'amount',
-                    'l10n_it_vat_due_date')
+                    'children_tax_ids')
```

Comment:
Should not this just be tax_tag_ids?  I suppose the exoneration on the group of taxes will be ignored anyways.

Reply:
The tax tag ids of the children taxes are involved, no?

Reply:
children_tax_ids.tax_tag_ids ? 🤔

Reply:
I remembered that we are no more checking the children but the tax itself, nvm, fixed.

Reply:
with `invoice_repartition_line_ids` and `refund_repartition_line_ids` it works